examples/04_GSD_FeI2.jl

```diff
@@ -141,18 +158,21 @@ end
 # this a real-space calculation, data is only available for discrete ``q`` modes
 # (the resolution scales like inverse system size).
 #
-# To store the correlation data, we initialize a `SampledCorrelations` object by
-# calling [`dynamical_correlations`](@ref). It requires three keyword arguments:
-# an integration step size, a target number of ωs to retain, and a maximum
-# energy ω to resolve. For the time step, twice the value used for the Langevin
-# integrator is usually a good choice.
+# The function [`dynamical_correlations`](@ref) creates an object to store
+# sampled correlations. The integration timestep `Δt` used for measuring
+# dynamical correlations can be somewhat larger than that used by the Langevin
+# dynamics. We must also specify `nω` and `ωmax`, which determine the
+# frequencies over which intensity data will be collected.
 
-sc = dynamical_correlations(sys_large; Δt=2Δt, nω=120, ωmax=7.5)
```

Comment:
I see the two is left here.

From an interface standpoint, I guess it's a bit confusing, because in the typical use case the user never sets a time step for ImplicitMidpoint. But they have the option of estimating if they so choose, and that estimate will be half of what is used here.